lib/fabrik/DirectorManager.js

```diff
@@ -194,9 +194,7 @@ class DirectorManager extends BaseManager {
             .getDeploymentManifest(deploymentName)
             .then(manifest => _
               .assign(opts, {
-                previous_manifest: manifest,
-                organization_guid: _.get(previousValues, 'organization_id'),
-                space_guid: _.get(previousValues, 'space_id')
+                previous_manifest: manifest
```

Comment:
You seem to have removed orgid & space id.  If you are removing org id and space id, always ensure context is passed so that consumers (service manifests) can pick the same from context. You are passing it as part of tags, but explicitly passing in context & calling it as context makes sense from readability perspective of services manifest, as it does not seem right that one picks up org id from tags. This way it remains consistent across all consumers. Everyone lookups context object to get what they want, rather than looking up tags in some place and context in other.

Reply:
Added opts.context in tags, but it is not sufficient for getSchedule update api. 

but whye 
`
-                organization_guid: _.get(previousValues, 'organization_id'),	
-                space_guid: _.get(previousValues, 'space_id'
`required. Needs to be verified.

Reply:
org_guid and space_guid has to be passed, as generateManifest is being called from diffManifest also.
Or in generateManifest, we can check if `opts.context` is null then assign tags to org_guid and space_guid
https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/84702b2fc2c2cf86706fcd1c88c612810a41d172/lib/jobs/ServiceInstanceUpdateJob.js#L121

Reply:
That part, I would take care.
How about  following?
'
organization_guid: _.get(previousValues, 'organization_id'),	
space_guid: _.get(previousValues, 'space_id'
'